backend/src/main/java/io/debezium/configserver/model/JolokiaURIsList.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.configserver.model;
+
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collection;
+
+@Schema(implementation = String.class, type = SchemaType.ARRAY, example = "[\"http://localhost:1234\"]")
+public class JolokiaURIsList extends ArrayList<URI> {
```

Comment:
Are you sure this is needed? A similar construct in another part of the code was only required for the `@Schema` annotation. This should not be needed in this case. A simple `List<URI>` should do well.